.github/workflows/ci.yaml

```diff
@@ -592,9 +600,62 @@ jobs:
           projectToken: 695c25b6cb65
           workingDir: "./site"
 
+  offlinedocs:
+    name: offlinedocs
+    needs: changes
+    runs-on: ${{ github.repository_owner == 'coder' && 'buildjet-8vcpu-ubuntu-2204' || 'ubuntu-latest' }}
+    if: needs.changes.outputs.offlinedocs == 'true' || needs.changes.outputs.ci == 'true'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: ./.github/actions/setup-node
+        with:
+          directory: offlinedocs
+
+      - name: Setup Go
+        uses: ./.github/actions/setup-go
+
+      - name: Install go tools
+        run: |
+          go install github.com/golang/mock/mockgen@v1.6.0
+
+      - name: Setup sqlc
+        uses: sqlc-dev/setup-sqlc@v3
+        with:
+          sqlc-version: "1.19.1"
+
+      - name: Install dependencies
+        run: |
+          cd offlinedocs
+          yarn
+          # Install prettier globally
+          prettier_version=$(jq -r '.devDependencies.prettier' < package.json)
+          yarn global add prettier@$prettier_version
+
+      - name: Format
+        run: |
+          cd offlinedocs
+          yarn format:check
+
+      - name: Lint
+        run: |
+          cd offlinedocs
+          yarn lint
+
+      - name: Build
+        run: |
+          version="$(./scripts/version.sh)"
+          yarn global add prettier    # Install prettier before running the script
```

Comment:
Is this line necessary? You already install it globally above

Reply:
no. It was left out.